.github/workflows/ci.yml

```diff
@@ -41,6 +41,10 @@ jobs:
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
       run: bash rav1e.cmd
+    - name: Build SVT-AV1
+      if: steps.cache-ext.outputs.cache-hit != 'true'
```

Comment:
If I leave this line, then it seems that the "`bash svt.cmd`" command at line 47 is not executed.

If I delete this line, then the "`bash svt.cmd`" command is executed, but cmake fails to find NASM. Here is the output log:
https://github.com/AOMediaCodec/libavif/actions/runs/1758324173

The relevant excerpt is:

```
-- Looking for a ASM_NASM compiler
-- Looking for a ASM_NASM compiler - NOTFOUND
-- The ASM_NASM compiler identification is unknown
-- Didn't find assembler
CMake Error at CMakeLists.txt:72 (enable_language):
  No CMAKE_ASM_NASM_COMPILER could be found.

  Tell CMake where to find the compiler by setting either the environment
  variable "ASM_NASM" or the CMake cache entry CMAKE_ASM_NASM_COMPILER to the
  full path to the compiler, or to the compiler name if it is in the PATH.
```